usaspending_api/api_docs/api_documentation/awards/idvs/activity.md

```diff
@@ -10,7 +10,7 @@ Returns IDVs or contracts related to the requested Indefinite Delivery Vehicle a
 - award_id: (required) ID of award to retrieve. This can either be `generated_unique_award_id` or `id` from awards table.
 - limit: (optional, default: 10) The number of records to return.
 - page: (optional, default: 1) The page number to return.
-- sort: (optional, default: `period_of_performance_start_date`) The field on which to sort results.  Can be one of: `award_type`, `description`, `funding_agency`, `last_date_to_order`, `obligated_amount`, `period_of_performance_current_end_date`, `period_of_performance_start_date`, or `piid`.
+- sort: (optional, default: `obligated_amount`) The field on which to sort results.  Can be one of: `award_type`, `description`, `funding_agency`, `last_date_to_order`, `obligated_amount`, `period_of_performance_current_end_date`, `period_of_performance_start_date`, or `piid`.
```

Comment:
The sort order looks hardcoded in the SQL, perhaps `sort` should be removed from the API doc

Reply:
Thanks. I removed the sort documentation.